daemon/rename.go

```diff
@@ -103,7 +103,10 @@ func (daemon *Daemon) ContainerRename(oldName, newName string) (retErr error) {
 			container.Name = oldName
 			container.NetworkSettings.IsAnonymousEndpoint = oldIsAnonymousEndpoint
 			if e := container.CheckpointTo(daemon.containersReplica); e != nil {
-				log.G(context.TODO()).Errorf("%s: Failed in writing to Disk on rename failure: %v", container.ID, e)
+				log.G(context.TODO()).WithFields(logrus.Fields{
+					"containerID":   container.ID,
+					logrus.ErrorKey: e,
+				}).Error("failed to write container state to disk during rename")
```

Comment:
We're trying to reduce _direct_ uses of `logrus` (and where possible use containerd's `log` package, which also provides aliases for most things);

- can you use `log.Fields`
- for `logrus.ErrorKey`, it's fine to use "error" instead (although the `log` package _may_ have an alias; I don't think we really need that though)